fixtures/act/src/index.test.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import TestUtils from 'react-dom/test-utils';
+import TestRenderer from 'react-test-renderer';
```

Comment:
Do these fixture tests run when you run `yarn test`? If not, why are they in here? Can we move them to `packages/`?

Reply:
Our setup doesn't support multiple renderers in the same file without also resetting shared React module state (which would break the warning).

Reply:
^ hence this fixture. open to ideas, but this seemed cleanest (for something that's probably an edge case)

Reply:
Sounds like a problem with the jest config. At the very least can we run this in CI? I will never remember to run this command manually.

Reply:
Not sure what you mean by a problem with jest config. It doesn’t support so intentionally because of how we mock the inlined host configs.

Reply:
I've updated the PR to run these tests on ci